src/display/api.js

```diff
@@ -701,6 +701,8 @@ var PDFDocumentProxy = (function PDFDocumentProxyClosure() {
  *                                calling of PDFPage.getViewport method.
  * @property {string} intent - Rendering intent, can be 'display' or 'print'
  *                    (default value is 'display').
+ * @property {boolean} enableWebGL - (optional) Enables WebGL accelerated
```

Comment:
This might come in handy: http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values